test/functional/subscription_controller_test.rb

```diff
@@ -27,4 +27,14 @@ def setup
     assert_response :success
   end
 
+  test 'should subscribe to multiple tags' do
+    UserSession.create(users(:bob))
+    assert users(:bob).following(:awesome)
+    get :multiple_add, params: { type: 'tag', names: 'blog,kites,,balloon,awesome' }
+    assert_response :redirect
+    assert users(:bob).following(:blog)
```

Comment:
If a user checks any tag which is already subscribed then it must remain subscribed to the user.

Reply:
Here tag `awesome` status remains `following` for the user `bob`.